app/helpers/schools/placement_requests_helper.rb

```diff
@@ -34,10 +34,7 @@ def cancellation_noun(cancellation)
 
   def cancellation_reasons(cancellation)
     safe_join(
-      [
-        cancellation.humanised_rejection_category,
-        cancellation.reason
-      ].reject(&:blank?).map { |r| tag.p(r) }
+      Array.wrap(cancellation.humanised_rejection_categories).map { |r| tag.p(r) }
```

Comment:
There is a slight change here in that `reject(&:blank?)` has been removed. Are we no longer expecting empty cancellation reasons ? The equivalent would be:
``` Ruby
Array.wrap(cancellation.humanised_rejection_categories).compact.map { |r| tag.p(r) }
```

Reply:
Yeah so this method shifted in logic a bit and the thing that could be `blank` (the `reason`) was moved to the `humanised_rejection_categories` method, which now does the `reject(&:blank?)` so it should no longer be needed here.